modules/omnisearch/omnisearch-posts.php

```diff
@@ -71,13 +71,18 @@ function prepare_items() {
 	function column_post_title( $post ) {
 		$actions = array();
 		if ( current_user_can( $this->post_type_obj->cap->edit_post, $post ) ) {
+			$post_title = sprintf( '<a href="%s">%s</a>', esc_url( get_edit_post_link( $post->ID ) ), wptexturize( $post->post_title ) );
+		} else {
+			$post_title = wptexturize( $post->post_title );
+		}
+		if ( current_user_can( $this->post_type_obj->cap->edit_post, $post ) ) {
 			$actions['edit'] = sprintf( '<a href="%s">%s</a>', esc_url( get_edit_post_link( $post->ID ) ), esc_html( $this->post_type_obj->labels->edit_item ) );
```

Comment:
What do you think if we moved this line into the previous `if` since it's the same condtion?